packages/wonder-blocks-modal/components/modal-header.test.js

```diff
@@ -52,4 +58,28 @@ describe("ModalHeader", () => {
         // Assert
         expect(wrapper.exists()).toBe(true);
     });
+
+    test("renders the title by default", () => {
+        // Arrange
+        const wrapper = mount(
+            <ModalHeader
+                title="Title"
+                subtitle="Subtitle"
+                testId="test-example-header"
+                titleId="modal-title"
+            />,
+        );
+
+        // Act
+        const title = wrapper.find(
+            `[data-test-id="test-example-header-title"]`,
+        );
+        const subtitle = wrapper.find(
+            `[data-test-id="test-example-header-subtitle"]`,
+        );
+
+        // Assert
```

Comment:
It's generally good practice to have your assert section only assert for one thing - I think this test can be split into two - one to check the title and one to check the subtitle.